ui/components/NotificationCenter/metadata.js

```diff
@@ -104,7 +105,15 @@ const EmptyState = ({ event }) => {
       }}
     >
       {' '}
-      {event.description}{' '}
+      {
+        <TextWithLinks
```

Comment:
@aabidsofi19 is this desired?

Reply:
@ASK-03  you might use the `FormatStructuredData` Component directly

Reply:
yup , please try.

Reply:
In *metadata.js* file, the `FormattedMetadata` component checks if the event has metadata or not and then if it doesnot have metadata it returns the event in `EmptyState` component and thats how the notification for *Design Import* are processed (as they donot have metadata). I think adding a `TextWithLink` component is better as it will just only check the content of the notification and detect potential links.